### ISSUE TITLE
The theme does not work properly

### ISSUE THREAD

**Issue by dodancs** [joelacus/deluge-web-dark-theme#11]
![image](https://github.com/joelacus/deluge-web-dark-theme/assets/1970811/5f893a3a-927c-47a5-928c-4831fff9d8a7)

Followed the instructions, made sure the files are accessible by deluge-web (applied proper permissions and checked via network monitor in the browser that all assets are loading), and still, it is grey, but now it is broken.

**Comment by joelacus** [comment]
It looks like the new icons are loaded, but not the CSS file. The new icons are a higher resolution than the old ones, and the sizes are controlled by the new CSS theme, that's why it looks broken.
Is "xtheme-dark.css" listed under the stylesheets if you view the page source? 
Try forcing your browser to redownload all page assets with CTRL+SHIFT+R.

**Comment by eojjn** [comment]
hey, I'm in a similar position to dodancs and can confirm the CSS is not being loaded and is not in the stylesheets listed in page source...

**Comment by joelacus** [comment]
If the "xtheme-dark.css" file is not being loaded, then maybe the deluge web config is wrong.
Double check that "theme" in the "web.conf" file is set to "dark", it may have been reverted if deluge is still running or if a file called "web.conf~" exists (delete that if it does). Then restart deluge and refresh the page with CTRL+SHIFT+R.

If this doesn't help, let me know what version of Deluge you are using, and on what OS, and I'll try to replicate issue and find a solution for you.

**Comment by Aeqas** [comment]
I'm having the exact same issue. I double checked that there is only a web.conf (no web.conf~) and that the theme value is set to dark. No change after a restart. 

I'm using Deluge 2.1.1 and its running on Windows 11 Pro.

**Comment by joelacus** [comment]
Ah, I wonder if the others are using Windows. The web.conf~ is a Linux backup file, so that wouldn't be an issue. 
Strange that three people have the same issue at the same time after 2 years. I'll have a look into it for you.

**Comment by joelacus** [comment]
I've set up Windows 11 Pro in a virtual machine and installed Deluge 2.1.1 and the theme and it works fine.
I did notice however, that instead of "web.conf~" there is a "web.conf.bak" file, essentially the same. This may be reverting "web.conf" to the original version?

If the theme value is still set to "dark" though in web.conf, then try reinstalling the theme files again and double check that "xtheme-dark.css" has been copied into "C:\Program Files\Deluge\deluge\ui\web\themes\css".

If this still doesn't help then unfortunately I'm not sure what the problem could be :/
Reinstalling Deluge will restore the old icons.

**Comment by Aeqas** [comment]
I checked and I did have a "web.conf.bak" file. I edited it to also be set to "dark" instead of "gray" (after stopping the daemon and web services), but no luck. I also outright deleted the file, but no change.

I did a fresh install of the theme files as well since I had reverted it last night by swapping all the values and images back from a backup I had created, and confirmed that "xtheme-dark.css" was present in the correct place.

Maybe its an issue with how I installed Deluge. I'll see if I can reinstall it some time soon and see if I can get it to work as well.

**Comment by joelacus** [comment]
That's odd then, I'm sorry it isn't working for you. 
I installed "deluge-2.1.1-win64-setup.exe" and enabled WebUI from the plugins section in the settings. I don't know if there is another way, but if you figure it out, please let me know.

**Comment by Aeqas** [comment]
I figured it out and got it working. 

After sleeping on it I realized that you were running the webUI plugin, but I wasn't, yet I was still connecting to deluge remotely through the web console. I'm running deluge as a thinclient with a seperate deluge-web service, and looking through the instructions on how to get that service set up (https://deluge.readthedocs.io/en/latest/how-to/nssm-service.html), it points to C:\config_location. Inside this folder is *another* web.conf and web.conf.bak, updating these 2 files along with the 2 original files in appdata gets the theme to work. 

Hopefully this also helps the other 2.

![image](https://github.com/joelacus/deluge-web-dark-theme/assets/43660405/387fd9df-b9e9-4901-a57f-a93dcfcf0955)
